### PR TITLE
Restyle Homepage Location Section

### PR DESCRIPTION
**Pull request by jhancock532**
#### [Test these changes using GitPod](https://gitpod.io/#https://github.com/jhancock532/bakerydemo/tree/restyle-homepage-location-cards)

Refactors the location cards into a seperate card template, following the folder structure of previous MRs. Implements the location cards according to the designs.

#### [Link to designs on Figma](https://www.figma.com/file/pQXoIKqxSK4QmOAEadeSkd/Wagtail-Bakery?node-id=24%3A2865)

<details><summary>Screenshots</summary>
image
image
</details>

### CODE REVIEW

File: bakerydemo/templates/includes/card/location-card.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% load wagtailimages_tags %}
+
+<div class="location-card col-sm-4">
+    <a class="location-card__link" href="{{page.url}}">
+        <figure class="location-card__image">
+            {% image page.image fill-430x320-c100 %}
+        </figure>
+        <div class="location-card__contents">
+            <h3 class="location-card__title">{{page.title}}</h3>
+            <p class="location-card__text">{{ page.introduction|truncatewords:15 }}</p>
```
Comment (chris-lawton) [inline bakerydemo/templates/includes/card/location-card.html]: suggestion: wrap this in conditional if it's not a required field

File: bakerydemo/templates/base/home_page.html [hunk 2]
```diff
@@ -73,24 +73,12 @@ <h2>{{ page.promo_title }}</h2>
...
-                            <div class="feature-2-text">
-                                <h3><a href="{{childpage.url}}">{{childpage.title}}</a></h3>
-                                <p>{{ childpage.introduction|truncatewords:15 }}</p>
-                            </div>
-                        </li>
-                    {% endfor %}
-                </div>
+                <h2 class="locations-section__title">{{ page.featured_section_2_title }}</h2>
+                {% for childpage in page.featured_section_2.specific.children|slice:"3" %}
+                    {% include "includes/card/location-card.html" with page=childpage %}
```
Comment (chris-lawton) [inline bakerydemo/templates/base/home_page.html:80]: much neater 👍

File: bakerydemo/static/css/main.css [hunk 3]
```diff
@@ -1347,4 +1327,71 @@ input[type="radio"] {
...
+  color: var(--dark-orange);
+  text-decoration: underline;
+}
+
+.location-card__image {
+  margin-bottom: 0;
+  overflow: hidden;
+}
+
+.location-card__image img {
```
Comment (chris-lawton) [inline bakerydemo/static/css/main.css:1458]: suggestion: add `object-fit: cover` so the images don't appear stretched (not supported in ie11)

bakery-object-fit

Comment (jhancock532) [reply bakerydemo/static/css/main.css:1458]: Thank you, that's much better

**Pull request merged**